doc/migration_notes.rst

```diff
@@ -239,6 +239,40 @@ Removed features:
      * Use ``nrf54l15dk`` instead.
 
    * Native TLS support including ``overlay-native_tls.conf``.
+
+   * TCP and UDP clients.
+     This includes the removal of the following AT commands:
+
+      * ``AT#XTCPCLI``
+      * ``AT#XTCPSEND``
+      * ``AT#XUDPCLI``
+      * ``AT#XUDPSEND``
+
+      You can replace this functionality by using the socket AT commands.
+
+      Migration examples:
+
+        * TCP IPv4 client:
```

Comment:
In terms of style how this should be written, show a sequence of `AT#XTCPCLI` and `AT#XTCPSEND`, and how that looks with socket AT commands.